projects/elife.yaml

```diff
@@ -1687,6 +1687,24 @@ digests:
         ports:
             1290: 80
 
+
+elife-libero-reviewer:
+    description: replacement for xpub
+    default-branch: null
+    domain: null
+    intdomain: null
+    aws:
+        ec2: false
+    aws-alt:
+        staging:
+            rds:
+                engine: postgres # or 'MySQL'
+                version: '10.6'
```

Comment:
any flexibility in getting this bumped to postgresql 11 ? I'm aware the versions of mysql/psql that we're using are aging and will need to be upgraded soon.

Reply:
Yes. This will be part of our testing of xpub to reviewer migration.